### ISSUE TITLE
Predicted Contact Maps (no structure)

### ISSUE THREAD

**Issue by deboramarks** [debbiemarkslab/EVcouplings#18]
Add to couplings stage

**Comment by aggreen** [comment]
Currently the predicted contact maps for proteins with no structure are outputs of the compare stage. I know it seems somewhat illogical to run 'compare' and collect its outputs for a protein with no structure, but if we change this then we'll have contact maps in two different locations, which also seems illogical and will get confusing. Plus, the user won't necessarily know if they have a structure before they execute the pipeline.

**Comment by aggreen** [comment]
@thomashopf Debbie mentioned that you two had discussed this today - are there any special considerations?

**Comment by thomashopf** [comment]
Yes we discussed this a while ago when I was still in Boston... I made very much the same arguments as you, Anna, so I am not sure if this is worth having (then again, I doesn't cost us that much)
